db/model/Gdoc/htmlToEnriched.ts

```diff
@@ -596,6 +614,92 @@ function finishWpComponent(
                 }
             } else return { ...content, errors }
         })
+        .with("owid/summary", () => {
+            // Summaries can either be lists of anchor links, or paragraphs of text
+            // If it's a paragraph of text, we want to turn it into a callout block
+            // If it's a list of anchor links, we want to turn it into a toc block
```

Comment:
When a summary block is both text and list (e.g. https://ourworldindata.org/vaccination), both conditions get skipped below and the summary gets stripped out of the output.

Reply:
Yeah, we don't really have a way to render a similar component in Gdocs. Instead we'll track it and deal with it manually.